Fixing incident alerts and artifacts is not populated (#33558)
demisto/content: Packs/CortexXDR/ReleaseNotes/6_1_27.md

#### Integrations

##### Palo Alto Networks Cortex XDR - Investigation and Response

> - Fixed an issue where ***alerts***, ***network_artifacts***, ***file_artifacts*** is not being populated on new incidents when bringing them into XSOAR.

Reviewer: Add rn regarding other fixes in this version

##### Palo Alto Networks Cortex XDR - Investigation and Response

- Fixed an issue where ***alerts***, ***network_artifacts***, ***file_artifacts*** is not being populated on new incidents when bringing them into XSOAR.

> - Fixed an issue which alert_limit_incident isn't set when docker has a timeout and incidents were not added yet.

Reviewer: ```suggestion
- Fixed an issue where the alert count was reset after docker timeout.
```

> - Fixed an parsing the response's format differed from the origin after improving implementation.

Reviewer: ```suggestion
- Fixed a bug where the response from the API was not parsed correctly.
```
